src/backend/joanie/core/api/client.py

```diff
@@ -30,6 +31,7 @@
 from joanie.core.tasks import generate_zip_archive_task
 from joanie.core.utils import contract as contract_utility
 from joanie.core.utils import contract_definition, issuers
+from joanie.core.utils.payment_schedule import generate
```

Comment:
```suggestion
from joanie.core.utils import payment_schedule
```

And then use `payment_schedule.generate` ? Otherwise, reading where `generate` is called, it's not really easy to understand what this function is doing.

Reply:
I used `from joanie.core.utils.payment_schedule import generate as generate_payment_schedule` for clarity